index.html

```diff
@@ -1,26 +1,126 @@
-<doctype html>
-<html>
-
-<head>  
-<title> </title>
-<style type="text/css">
-	
-</style>
-
-</head>
-
-<body>
-
-
-
-
-
-
-<p>Hello world</p>
-
-
-</body>
-
-</html>
-
-
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Youtube Video Page</title>
+    <link type="text/css" rel="stylesheet" href="stylesheet.css" />
+  </head>
+  <body>
+    <div>
+      <nav class="navbar">
+        <div class="menu">
+          <button class="menu_button"><span><img src="images/menu-icon.png" alt="menu icon" /></span></button>
+        </div>
+        <div class="logo">
+          <a href="#"><img src="images/youtube-icon.png" height="25" width="80"  alt="youtube icon" /></a>
+        </div>
+        <div class="search-div">
+          <form>
+            <input type="text" name="search" class="input1" placeholder="Search" />
+            <button class="input2"><img src="images/search-icon.png" alt="search icon" /></button>
+          </form>
+        </div>
+        <div class="submenu">
+          <ul>
+            <li><a href="#"><img src="images/video-icon.jpg" alt="video-icon" /></a></li>
+            <li><a href="#"><img src="images/google-menu-icon.png" alt="google menu icon" /></a></li>
+            <li><a href="#"><img src="images/notification-icon.jpg" alt="notification icon" /></a></li>
+            <li><a href="#"><img src="images/a-icon.jpg" class="a-icon" alt="profile icon" /></a></li>
+          </ul>
+        </div>
+        
+        
+      </nav>
+      <main class="main_section">
+        <section class="main">
+          <div >
+            <iframe class="vid" src="https://www.youtube.com/embed/Ve8swq7lisU" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Could you remove the `frameborder` attribute and use CSS instead. The equivalent should be styling the iframe element as `border: none;`

Learn more here https://wiki.whatwg.org/wiki/Presentational_elements_and_attributes

Reply:
Thanks for the review.